Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM node:8.16-alpine as builder
+
+# Install SSL ca certificates
+#RUN apk update && apk add git && apk add ca-certificates
+
+# Create appuser
+#RUN adduser -D -g '' appuser && mkdir /go-analyzer
+RUN mkdir /javascript-analyzer
```

Comment:
Since the `WORKDIR` command is used later, the `RUN mkdir` can be removed - `WORKDIR` creates the directory if it does not exist